src/app/app.html

```diff
@@ -12,7 +12,7 @@
           {{p.title}}
         </button>
       </ion-list>
-      <div style="position: absolute; bottom: 1%; margin-left: 5%;" *ngIf="showNativeStoreAd">
+      <div style="position: absolute; bottom: 1%; margin-left: 5%;" *ngIf="runningInBrowser">
```

Comment:
Eh, doesn't hurt to leave it.  It's not in the way of anything, and maybe it'll get some clicks.